examples/nextjs-comments-overlay/src/styles/comments.css

```diff
@@ -0,0 +1,23 @@
+:root {
+  --lb-accent: rgb(var(--color-brand));
+}
+
+.lb-thread,
+.lb-composer {
+  border-radius: var(--radius);
+}
+
+.lb-composer {
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+}
+
+.lb-tooltip,
+.lb-dropdown-item,
+.lb-composer-suggestions {
+  z-index: var(--z-above-comments-sidemenu);
+}
+
+.lb-portal {
+  z-index: var(--z-above-comments-sidemenu) !important;
```

Comment:
`!important` won't be necessary when the CSS improvements land in the next release.